src/server/transaction/server/testing/server_test.go

```diff
@@ -337,51 +314,30 @@ func TestTransactions(suite *testing.T) {
 		requireCommitResponse(t, info.Responses[2], commitE)
 		requireEmptyResponse(t, info.Responses[3])
 
-		commitInfos, err := env.PachClient.ListCommitByRepo(client.NewRepo("A"))
+		commitInfos, err = env.PachClient.ListCommitByRepo(client.NewRepo("A"))
 		require.NoError(t, err)
-		require.Equal(t, 1, len(commitInfos))
+		require.Equal(t, aCommits+1, len(commitInfos))
```

Comment:
Because of alias commits, we actually end up with several commits in each branch, so this is just making the check that the transaction adds exactly one commit to each branch (which is what this test was intended to check).